### ISSUE TITLE
训练MaskRCNN的过程中出现CUDA Out of memory，且显存未正常释放

### ISSUE THREAD

**Issue by Renzhihan** [WZMIAOMIAO/deep-learning-for-image-processing#718]
**System information**
* Have I written custom code: No
* OS Platform(e.g., window10 or Linux Ubuntu 16.04): Ubuntu18.104
* Python version: 3.7
* Deep learning framework and version(e.g., Tensorflow2.1 or Pytorch1.3): Pytorch 1.12.1
* Use GPU or not: GPU 2080ti
* CUDA/cuDNN version(if you use GPU): CUDA 10.2
* The network you trained(e.g., Resnet34 network): MaskRCNN 

**Describe the current behavior**
在双卡训练MaskRCNN的过程中突然出现CUDA Out of memory（bs设置为2），同时程序中止后第二张卡的显存也没有正常释放。
单卡设置bs为1也会出问题。
出错的地方为
**Error info / logs**
File "/home/mask_rcnn/network_files/transform.py", line 117, in _resize_image_and_masks
    mask[:, None].float(), size=size, scale_factor=scale_factor, recompute_scale_factor=recompute_scale_factor

**Comment by Renzhihan** [comment]
您好，经过检查我的数据集确实是太大了导致了OOM，但是程序中止后显存未正常释放的问题您有空了可以检查下是不是哪里有问题（不是每次OOM后都会出现）。
